chroma-manager/tests/integration/core/utility_testcase.py

```diff
@@ -230,9 +230,10 @@ def _fetch_help(self, assert_test, tell_who, message=None, callback=lambda: True
 
             # First create the file, errors in here do destroy the original, but will be reported by the test framework
             fd = os.open(key_file, os.O_RDWR | os.O_CREAT)
-            os.write(fd, "Subject: %s\n\n%s\n\nTest Runner %s" % (message, message, socket.gethostname()))
+            os.write(fd, "To: %s\nSubject: %s\n\n%s\n\nTest Runner %s" %
+                     (', '.join(tell_who), message, message, socket.gethostname()))
             os.lseek(fd, 0, os.SEEK_SET)
-            subprocess.call(['sendmail'] + tell_who, stdin=fd)
+            subprocess.call(['sendmail'] + ['-t'], stdin=fd)
```

Comment:
nit, but no need to concat two lists when their contents are effectively static.